### ISSUE TITLE
Undefined array key ""

### ISSUE THREAD

**Issue by LMS235** [lpaulsen93/dokuwiki-plugin-odt#276]
Error with PHP 8.1.2 (while indexing)

...
PHP Warning:  Trying to access array offset on value of type null in .../lib/plugins/odt/helper/config.php on line 578
PHP Warning:  Undefined array key "" in .../lib/plugins/odt/helper/config.php on line 561
...

**Comment by nerun** [comment]
Same warnings with PHP 8.1 and while indexing. But i found an easy fix.
# How to Easy Fix
## /odt/helper/config.php `line 561`:
Find:
```php
if (!$value && $conf[$dw_name]) {
```
Change to:
```php
if (!$value && isset($conf[$dw_name])) {
```
## /odt/helper/config.php `line 578`:
Find:
```php
$value = $odt_meta[$name];
```
Change to:
```php
if(isset($odt_meta[$name])) $value = $odt_meta[$name];
```

**Comment by nerun** [comment]
@lpaulsen93 it's fixed.

**Comment by Klap-in** [comment]
In my understanding this was fixed by #283 which included the referred PR.
